env/includes/settings/tk-nuke-writenode.yml

```diff
@@ -36,7 +36,7 @@ settings.tk-nuke-writenode.shot:
   template_script_work: nuke_shot_work
   write_nodes:
   - file_type: exr
-    name: exr, 32 bit
+    name: exr, lossless, zip, 32 bit
```

Comment:
Algemene vraag, wat doet die lossless hier?

Reply:
Aangeven aan artist dat iets wat ze renderen lossless is of lossy, ietsje duidelijker maken. Was feedback van Ewoud keertje